src/Features/Core/Portable/Diagnostics/Service/DiagnosticAnalyzerService.cs

```diff
@@ -122,8 +124,22 @@ public async Task<ImmutableArray<DiagnosticData>> GetDiagnosticsForSpanAsync(
             document, range, shouldIncludeDiagnostic, priorityProvider, diagnosticKinds, cancellationToken).ConfigureAwait(false);
     }
 
-    public Task<ImmutableArray<DiagnosticData>> ForceAnalyzeProjectAsync(Project project, CancellationToken cancellationToken)
-        => _incrementalAnalyzer.ForceAnalyzeProjectAsync(project, cancellationToken);
+    public async Task<ImmutableArray<DiagnosticData>> ForceAnalyzeProjectAsync(Project project, CancellationToken cancellationToken)
```

Comment:
Might already know but this one includes the changes from here: https://github.com/dotnet/roslyn/pull/79983. So there's probably some merge fun to be had with this when the other is merged.

Reply:
@CyrusNajmabadi is building off the commits from the other PR, so Git won't care as long he doesn't squash the first PR.